PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -1553,7 +1553,7 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			else if (wParam == NPPMAINMENU)
 				return (LRESULT)_mainMenuHandle;
 			else
-				return NULL;
+				return static_cast<LRESULT>(NULL);

```

Comment:
`LRESULT` window procedure function usually return `0` when processing some message. For example https://docs.microsoft.com/en-us/windows/win32/gdi/wm-paint#return-value